balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -18,7 +18,7 @@
  * NOTE: Copied and Pasted from another existing project.
  * TODO: Deprecate this Concurrent Modification Exception prone class.
  */
```

Comment:
Since this is a pretty central class, and the name doesn't provide much insight into it's true nature (it's both a consumer and a producer, consuming from local files, sending to a remote AMQ queue), a quick summary here saying how it runs as a separate thread and reads metrics files from the files in the directory tree, buffering them before sending them on to AMQ.